group/src/main/java/com/android/group/view/recyclerview/AddLocationViewHolder.java

```diff
@@ -1,30 +1,38 @@
 package com.android.group.view.recyclerview;
 
-import android.content.DialogInterface;
 import android.support.annotation.NonNull;
 import android.support.v7.app.AlertDialog;
 import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
+import android.widget.Button;
 import android.widget.ImageView;
 import android.widget.TextView;
 
 import com.android.group.R;
 import com.android.group.model.Venue;
+import com.android.group.model.yelp.Business;
 import com.android.group.view.OnFragmentInteractionCompleteListener;
 import com.android.group.viewmodel.NetworkViewModel;
+import com.android.interactionlistener.OnFragmentInteractionListener;
 import com.squareup.picasso.Picasso;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class AddLocationViewHolder extends RecyclerView.ViewHolder {
 
     private ImageView venueImageView;
```

Comment:
try to have these views local to the method that assigns things to them

Reply:
All set.